### ISSUE TITLE
[Bug] Anchors

### ISSUE THREAD

**Issue by Mystqr4** [WindowTop/WindowTop-App#325]
I have noticed that at times anchors will appear for no reason such as when I fullscreen a video in MPC-HC, software like Open Hardware Monitor's widget will become an anchor and hide the widget even if I have set it to be always on top.  It did the same thing in MPC-BE and Potplayer in fullscreen. Interestingly when it's not set to always on top it will not create and anchor overlay on fullscreen in these players.

I first experienced this bug in beta 10 or 11 with FXsound upon boot then it continued in the final but dismissed it as an odd occurrence and put it at the back of my mind but have found that it is fairly consistent and easy to reproduce with the above video players in fullscreen at least.

I wasn't able to reproduce this bug with VLC, SMplayer or Gom player in fullscreen.

**Comment by gileli121** [comment]
@Mystqr4 
I checked your repord.
I could reproduce it using MPC-HC.

But this is not a bug. The old behavior was a bug.
I detected an issue that anchors may not always be on top because there was a problem in previous versions, that a code that should execute was not always executed.
It was fixed. This is by design. The idea of anchors is that they will be even above the top-most windows.

However, I do see your point. If you do a fullscreen, I see valid point here to auto-hide anchors. 

There is open enhancement here:

https://github.com/WindowTop/WindowTop-App/issues/239

The idea here is to hide anchors when the mouse is hidden, and this happens when you do a fullscreen and do more stuff like typing...

Anyway, I suggest setting a hotkey to hide the anchors. you can configure the hotkey here
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/c947cbc6-5bb2-4f43-bfe7-2b641c08f0b5)
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/38e34c68-ea71-449e-b3b7-8d45307b2c01)


Another thing I noticed is that if I use PotPlayer instead of MPC-HC, I don't get this behavior, and anchors are left behind. So maybe you can consider switching to PotPlayer.
I noticed that the author of MPC-HC wrote that the project has been dead since 2017 and is no longer under development. So I think that you should probably switch anyway.

**Comment by Mystqr4** [comment]
Very odd... I tried it today after suspending my laptop and it doesn't appear to be doing it anymore which is very odd. I also experienced this same behavior with potplayer too  but it appears it is not a problem for your version of Potplayer (mine is a dew version behind)

That is indeed true however MPC-HC has long since been forked by clsid2 and is constantly under development as much as both PotPlayer and MPC-BE.

I do like the idea of having an option to hide anchors when your mouse pointer disappears, it seems fairly logical since these are the moments you probably do not want anchors all over your screen and then by moving your mouse the anchors could appear again. I will keep an eye on it has oddly stopped doing it now. I did notice it would do it on boot fro software like FXsound without me touching the mouse at all or the FXsound software being visible or on top but rather set to open in the tray on startup. It just seemed odd. I will keep an eye on it.
Another solution is for me to Blacklist Open Hardware Monitor too if need be.

Here is the updated fork of MPC-HC
https://github.com/clsid2/mpc-hc/releases

There is a much more up to date fork of Open Hardware Monitor called Libre Hardware Monitor for anyone reading which is still actively being developed too.
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor

There might be something about suspending your computer that causes problems for WindowTop. In some rare occasions if I close the lid on my laptop WindowTop sometimes disappears from my processes and I have to open it up again. It rarely happens but I thought I would mention it since suspending my laptop seems to have changed the behaviour of WindowTop this time to not show an anchor for the Open Hardware Monitor Desktop 'Gadget'.

I do appreciate you having taken the time to look at it anyway.
